lib/exception_handling.rb

```diff
@@ -214,21 +203,41 @@ def write_exception_to_log(ex, exception_context, timestamp)
       end
     end
 
+    #
+    # Send notifications to configured external services
+    #
+    def send_external_notifications(exception_info)
+      results = {}
+      if honeybadger?
+        results[:logged_to_honeybadger] = send_exception_to_honeybadger(exception_info)
+      end
+
+      if should_send_email?
+        log_error_email(exception_info)
```

Comment:
since we're doing it this way, do we also want to store `email_sent` to pass to the callback function?

Reply:
hmm, you're right. lets punt for now then to get this rca item addressed.